mx.truffleruby/mx_truffleruby.py

```diff
@@ -19,7 +19,7 @@
     import mx_truffleruby_benchmark
 
 _suite = mx.suite('truffleruby')
-rubyDists = ['TRUFFLERUBY', 'TRUFFLERUBY-TEST']
+root = _suite.dir
```

Comment:
`mx.suite('sulong')` does not work, but we could just check if `../sulong/mx.sulong` exists.